library/std/src/path.rs

```diff
@@ -2087,9 +2087,12 @@ impl Path {
     /// assert!(path.starts_with("/etc"));
     /// assert!(path.starts_with("/etc/"));
     /// assert!(path.starts_with("/etc/passwd"));
-    /// assert!(path.starts_with("/etc/passwd/"));
+    /// assert!(path.starts_with("/etc/passwd/")); // extra slash(es) is okay
     ///
     /// assert!(!path.starts_with("/e"));
+    /// assert!(!path.starts_with("/etc/passwd.txt"));
```

Comment:
Is this helpful? It seems pretty obvious just from the function name.

Reply:
Maybe a better example would be `!Path::new("/etc/passwd.txt").starts_with("/etc/passwd")`.

Reply:
I added that separately, see the assert below this. `assert!(!Path::new("/etc/foo.rs").starts_with("/etc/foo"));`